app/javascript/channels/client_channel.js

```diff
@@ -0,0 +1,20 @@
+export function getChannelName(url) {
+  return { channel: "ClientChannel", id: url.match(/\/clients\/(\d*)\/messages/)[1] }
```

Comment:
was this autogenerated? if not, might be helpful to either link to an explanation of this regex or add comments to explain. Is this the recommended approach from Rails documentation?